src/api-binder.ts

```diff
@@ -354,7 +354,16 @@ export class APIBinder {
 					'Trying to start poll without initializing API client',
 				);
 			}
-			this.pollTargetState(true);
+			this.config
+				.get('instantUpdates')
+				.catch(() => {
+					// Default to skipping the initial update if we couldn't fetch the setting
+					// which should be the exceptional case
+					return false;
```

Comment:
The current behavior if we can't fetch it is that we never poll the target state at all, so this is definitely an improvement in the exceptional circumstance that we do hit it